### PR TITLE
[build] Added cleanup after android/ubuntu build @open sesame 09/19 12:31

### PR DESCRIPTION
**Pull request by kparichay**
Added cleanup of patches after android/ubuntu build
Changed relative path of files to patches in patch file

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: api/android/build-android-lib.sh [hunk 1]
```diff
@@ -71,11 +71,10 @@ echo "NNStreamer root directory: $NNSTREAMER_ROOT"
 echo "Start to build NNStreamer library for Android."
 
 # Modify header for Android
-cd $NNSTREAMER_ROOT/api/capi
+cd $NNSTREAMER_ROOT
```
Comment (jaeyun-jung) [inline api/android/build-android-lib.sh]: let's add ```check_package patch``` in line 53

Comment (kparichay) [reply api/android/build-android-lib.sh]: done

File: meson.build [hunk 2]
```diff
@@ -230,6 +230,16 @@ endif
 
 # Patch for non-tizen build
 patch_file = join_paths(meson.current_source_dir(), 'packaging', 'non_tizen_build.patch')
+if (not get_option('enable-tizen')) and get_option('enable-capi')
+  r = run_command('patch', '-R', '--dry-run', '-sfp1', '-i', patch_file)
```
Comment (jaeyun-jung) [inline meson.build]: please remove run_command patch in ```/api/capi/meson.build```

if (get_option('enable-tizen'))
  message('CAPI is in Tizen mode')
  tizen_deps = [
    dependency('mm-resource-manager'),
    dependency('mm-camcorder'),
    dependency('capi-privacy-privilege-manager'),
    dependency('capi-base-common'),
    dependency('capi-system-info'),
    dependency('dlog')
  ]
**else
  r = run_command('patch', '-R', '--dry-run', '-sfp1', '-i', patch_file)
  if (r.returncode() != 0)
    r = run_command('patch', '-p1', '-i', patch_file)
    if (r.returncode() != 0)
      message('Non-Tizen mode support failed')
    endif
  endif
  message('CAPI is in non-Tizen mode')**
endif

Comment (kparichay) [reply meson.build]: My mistake, updated

**Pull request merged**